topdown/trace.go

```diff
@@ -44,14 +44,23 @@ const (
 	IndexOp Op = "Index"
 )
 
+// VarMetadata provides some user facing information about
+// a variable in some policy.
+type VarMetadata struct {
+	Name     string        `json:"text"`
```

Comment:
Nit: Can the JSON field be `"name"` instead of `"text"`?

Reply:
Yep, will do.